registry.k8s.io/manifests/k8s-staging-etcd/promoter-manifest.yaml

```diff
@@ -8,7 +8,7 @@
 # google group for gcr.io/k8s-staging-etcd is k8s-infra-staging-etcd@kubernetes.io
 
 registries:
-- name: gcr.io/k8s-staging-etcd
+- name: gcr.io/etcd-development
```

Comment:
The registry is `gcr.io/etcd-development/etcd`, I assume the suffix `/etcd` will automatically added somewhere.

Reply:
Should be coming from here:
https://github.com/kubernetes/k8s.io/blob/2c5d008075248a60d74504577ccc09df84aaa185/registry.k8s.io/images/k8s-staging-etcd/images.yaml#L1